content/codespaces/developing-in-codespaces/default-environment-variables.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Codespaces Default Environment Variables
+intro: 'GitHub sets default environment variables for each codespace.
+versions:
+  fpt: '*'
+  ghec: '*'
+type: overview
+topics:
+  - Codespaces
+  - Developer
+  - 
+product: '{% data reusables.gated-features.codespaces %}'
+---
+
+## About default environment variables
+
+{% data variables.product.prodname_dotcom %} sets default environment variables for every Codespace. Environment variables are case-sensitive. Commands run in codespaces can create, read, and modify environment variables.
+
+## List of default environment variables
+
+| Environment variable | Description |
+| ---------------------|------------ |
+| `CODESPACE_NAME` | Name of the codespace For example:`monalisa-github-hello-world-2f2fsdf2e` |
+| `CODESPACES` | Always `true` while in a codespace |
+| `GITHUB_TOKEN` | undefined |
+| `GIT_COMMITTER_NAME` | undefined |
+| `GIT_COMMITTER_EMAIL` | undefined |
```

Comment:
For these I think we should either give them descriptions or leave them off this list.

Reply:
yep, this was just a draft. I'll fill these out.